docs/diagrams/ParseAddProgressSequenceDiagram.puml

```diff
@@ -6,7 +6,7 @@ participant ":LogicManager" as LogicManager LOGIC_COLOR
 participant ":TutorAidParser" as TutorAidParser LOGIC_COLOR
 participant ":AddCommandParser" as AddCommandParser LOGIC_COLOR
 participant ":AddProgressCommandParser" as AddProgressCommandParser LOGIC_COLOR
-participant ":AddProgressCommand" as AddProgressCommand LOGIC_COLOR
+participant "a:AddProgressCommand" as AddProgressCommand LOGIC_COLOR
```

Comment:
I think nameless is better, don't really like the return arrows with the letter `a` just cos it doesn't have much meaning

Reply:
Removed :D